src/test/java/org/assertj/core/api/char2darray/Char2DArrayAssert_isDeepEqualTo_Test.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2020 the original author or authors.
+ */
+package org.assertj.core.api.char2darray;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.BDDAssertions.then;
+import static org.assertj.core.error.ShouldHaveSameSizeAs.shouldHaveSameSizeAs;
+import static org.assertj.core.error.ShouldNotBeNull.shouldNotBeNull;
+import static org.assertj.core.error.SubarraysShouldHaveSameSize.subarraysShouldHaveSameSize;
+import static org.assertj.core.error.array.MultidimensionalArrayShouldBeEqual.shouldBeEqual;
+import static org.assertj.core.presentation.StandardRepresentation.STANDARD_REPRESENTATION;
+import static org.assertj.core.util.AssertionsUtil.expectAssertionError;
+
+import org.assertj.core.api.Char2DArrayAssert;
+import org.assertj.core.description.EmptyTextDescription;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for <code>{@link Char2DArrayAssert#isDeepEqualTo(char[][])}</code>.
+ *
+ * @author Maciej Wajcht
+ */
+@DisplayName("Char2DArrayAssert isDeepEqualTo")
+class Char2DArrayAssert_isDeepEqualTo_Test {
+
+  @Test
+  void should_pass_if_both_actual_and_expected_are_null() {
+    // GIVEN
+    char[][] actual = null;
+    char[][] expected = null;
+    // WHEN/THEN
+    assertThat(actual).isDeepEqualTo(expected);
```

Comment:
replace by `assertThat` by `then` (from BDDAssertions) in all tests

Reply:
done